src/constants/coreSettings.ts

```diff
@@ -744,5 +744,37 @@ export const CORE_SETTINGS: SettingParams[] = [
     type: 'boolean',
     defaultValue: true,
     versionAdded: '1.10.5'
+  },
+  {
+    id: 'Comfy.ModelLibrary.AllowedSources',
+    category: ['Comfy', 'Model', 'Download'],
+    name: 'Model Download Allowed Sources',
+    type: 'hidden',
+    defaultValue: [
+      'https://civitai.com/',
+      'https://huggingface.co/',
+      'http://localhost:'
```

Comment:
should this be added only if current domain is localhost ? It would be safer no ?